chain/client/src/client_actor.rs

```diff
@@ -989,6 +989,9 @@ impl ClientActor {
                 warn!(target: "client", "receive bad block: {}", err);
             }
             Err(ref err) if err.is_error() => {
+                if let near_chain::ErrorKind::DBNotFoundErr(msg) = err.kind() {
+                    debug_assert!(!msg.starts_with("BLOCK HEIGHT"), "{:?}", err);
```

Comment:
why is this not possible? `DBNotFoundErr` can originate from many different places

Reply:
assert only for "BLOCK HEIGHT" column